src/Commands/InstallCommand.php

```diff
@@ -37,19 +37,21 @@ protected function nativePhpPath()
 
     protected function installNpmDependencies()
     {
-        $this->executeCommand('npm set progress=false && npm install', $this->nativePhpPath());
-    }
-
-    protected function executeCommand($command, $path)
-    {
-        $process = (Process::fromShellCommandline($command, $path))->setTimeout(null);
-
-        if ('\\' !== DIRECTORY_SEPARATOR && file_exists('/dev/tty') && is_readable('/dev/tty')) {
-            $process->setTty(true);
-        }
-
-        $process->run(function ($type, $line) {
-            $this->output->write($line);
-        });
+        $phpBinPackageDir = config('nativephp.binary_package');
+        $nativeBinaryPath = $phpBinPackageDir . 'bin/' . (PHP_OS_FAMILY === 'Windows' ? 'win' : 'mac');
+        $this->info("NativePHP binary path: $nativeBinaryPath");
+        $this->info('Fetching latest dependencies…');
+        Process::path(__DIR__ . '/../../resources/js/')
+                ->env([
+                    'NATIVEPHP_PHP_BINARY_PATH' => base_path($nativeBinaryPath),
+                    'NATIVEPHP_CERTIFICATE_FILE_PATH' => base_path($phpBinPackageDir . 'cacert.pem'),
+                ])
+                ->forever()
+                ->tty(PHP_OS_FAMILY != 'Windows')
+                ->run('npm install', function (string $type, string $output) {
```

Comment:
Should this be reverted to being the 'quiet' install? 

The non-quiet version helped solve a few issues when getting this going.

Would it be acceptable to be a bit more verbose during the Alpha?

Reply:
There's the commented out "isVerbose" if condition, which already checks for that. So if you run `php artisan native:install -v` you would get the verbose output.